### PR TITLE
Use ~/.aws/credentials file to login to ec2

### PR DESCRIPTION
**Pull request by jeanlaurent**
This will use standard documented aws credentials file, when no cli/env flag is passed.

This allow seamless login with people using other aws tools, as described in this [aws blog post](http://blogs.aws.amazon.com/security/post/Tx3D6U6WSFGOK2H/A-New-and-Standardized-Way-to-Manage-Credentials-in-the-AWS-SDKs)

### CODE REVIEW

File: drivers/amazonec2/amazonec2_test.go [hunk 1]
```diff
@@ -240,34 +240,10 @@ func TestSetConfigFromFlags(t *testing.T) {
...
-					{
-						AttributeName: &defaultVpc,
-						AttributeValues: []*ec2.AccountAttributeValue{
-							{AttributeValue: &vpcName},
-						},
-					},
-				},
-			},
-		}
+		return &faceEC2WithLogin{}
```
Comment (nathanleclaire) [inline drivers/amazonec2/amazonec2_test.go]: I think it should be `fakeEC2WithLogin{}`?

**Comment by nathanleclaire** [comment]
Looks like one small typo but other than that LGTM.

**Comment by nathanleclaire** [comment]
Nice job btw.  This is a very heavily requested user feature.

**Comment by jeanlaurent** [comment]
Corrected typo in code & docs, added readability to the tests, added simple IT test.

**Comment by dgageot** [comment]
LGTM

**Pull request merged**